File: UI/etravel_mobile/lib/models/paypal_link.dart
import 'package:json_annotation/json_annotation.dart';

part 'paypal_link.g.dart';

@JsonSerializable()
class PayPalLink {
  final String href;
  final String rel;
  final String method;

  PayPalLink({
    required this.href,
    required this.rel,
    required this.method,
  });

  factory PayPalLink.fromJson(Map<String, dynamic> json) =>
      _$PayPalLinkFromJson(json);

  Map<String, dynamic> toJson() => _$PayPalLinkToJson(this);
}

File: eTravelAgencija.Model/ResponseObject/PayPalLink.cs

namespace eTravelAgencija.Model.ResponseObject
{
    public class PayPalLink
    {
        public string Href { get; set; } = string.Empty;
        public string Rel { get; set; } = string.Empty;   // approve
        public string Method { get; set; } = string.Empty;
    }


}
